src/components/AssetsFilters/AssetsFilters.test.jsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 import { AssetsFilters } from './index';
 
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
I think you only need to configure this once so you can move this `Adapter` setup into a `setupTest.js` file and thus you don't have to call this at the beginning of all of the test files.